docs/development.rst

```diff
@@ -0,0 +1,102 @@
+Development
```

Comment:
Please have a look at the repository layout guideline, we use development_guide for these sort of information.